src/scala/io/bazel/rules_scala/scrooge_support/Compiler.scala

```diff
@@ -128,7 +128,7 @@ class Compiler {
           defaultOptional = isJava,
           skipIncludes = false,
           documentCache
-        )(com.twitter.logging.NullLogger) // scrooge warns on file names with "/"
```

Comment:
I removed it because I'm running a newer version of scrooge than 4.6.0 (4.15.0, via my project's WORKSPACE).  Everything works fine with the newer version except this:  in 4.7, scrooge [removed its dependency on twitter.util](https://github.com/twitter/scrooge/commit/de7e1d1c826d1549f6992ce5d68c444bc0720563), so this is java Logger now.  Removing this was the simplest way to make Compiler work with both versions of scrooge.

I can split this into a separate PR, if you'd prefer -- I think I'll upgrade rules_scala's scrooge to 4.7.0 and change this to a null logging implementation in the new interface, so it will work with newer versions as well.  Does that sound good?